packages/web-components/src/dialog-body/dialog-body.styles.ts

```diff
@@ -0,0 +1,124 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import {
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontSizeBase500,
+  fontWeightRegular,
+  fontWeightSemibold,
+  lineHeightBase300,
+  lineHeightBase500,
+  spacingHorizontalS,
+  spacingHorizontalXXL,
+  spacingVerticalL,
+  spacingVerticalS,
+  spacingVerticalXXL,
+} from '../theme/design-tokens.js';
+
+/** Dialog Body styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  /* formerly .root */
+  :host {
+    background: ${colorNeutralBackground1};
+    box-sizing: border-box;
+    display: grid;
```

Comment:
You are using the `display()` utility above, but setting it to grid here.

Reply:
This was a late addition to align with the React implementation a bit more (but that has a 2-col grid). Switched to `display('grid')`.